src/Plugin.php

```diff
@@ -0,0 +1,34 @@
+<?php
+declare(strict_types=1);
```

Comment:
The `develop` branch is for CakePHP 3 which supports PHP 5.6, so we can't have any `declare` statements or any other feature/syntax not available on PHP 5.6.